### DISCUSSION TITLE
Obsolete issues

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3467] [General]
I suggest to close the issues in this repository which are no longer relevant due to ramp-down.

Only issues which are likely to be worked on should be left open.

**Comment by MikeMcC399** [comment]
This is still an open suggestion ...

**Comment by MikeMcC399** [comment]
@Ein-Tim 
It doesn't seem that any action has been taken to close obsolete issues.

**Reply by Ein-Tim** [reply]
Yeah, I will raise that to the team.
